### DISCUSSION TITLE
Future of Bootstrap-Vue?

### DISCUSSION THREAD

**Discussion by dpalcic** [bootstrap-vue/bootstrap-vue#6741] [General]
Will there be bootstrap-vue for VueJS 3, or is this great project at it's end?

**Comment by ccsv** [comment]
Don't know what is going on but they stopped really updating the Project in February they didn't even commit any pull request.

https://github.com/bootstrap-vue/bootstrap-vue/graphs/commit-activity

But they use to do monthly updates
https://bootstrap-vue.org/docs/reference/changelog

No updates on twitter since January
https://twitter.com/BootstrapVue/with_replies

It is not like nobody is paying them either they have a 14k budget for 5 people for part time edits.
https://opencollective.com/bootstrap-vue#category-BUDGET

I think it is best to ask message an member of the team and ask.

**Comment by ccsv** [comment]
So I looked around and it appears the Vue3 and Bootstrap 5 option is MDBootstrap

https://mdbootstrap.com/docs/b5/vue/

**Comment by Avadaro** [comment]
В спонсорах проекта сейчас в основном казино, которые уже получили для себя необходимый функционал, и Vue3 с Bootstrap 5 им не к чему. Возможно еще будет какая-то работа над ошибками, но в развитие проекта не верится.
